pkg/reconciler/pipelinerun/resources/pipelinerunresolution_test.go

```diff
@@ -3823,6 +3809,7 @@ func TestResolvePipelineRunTask_WithMatrix(t *testing.T) {
 		name string
 		pt   v1.PipelineTask
 		want *ResolvedPipelineTask
+		pst  PipelineRunState
```

Comment:
is this pst used in the tests? I cannot find it used in the 2 test cases

Reply:
No, I don't think it's relevant to these test cases since we aren't dealing with results, but have added  it with whole array result examples in the subsequent PR: https://github.com/tektoncd/pipeline/blob/c164f71ab448c4ac263d0113bf1b58d1c0199be3/pkg/reconciler/pipelinerun/resources/pipelinerunresolution_test.go#L3822-L#3888